client/src/test/java/com/vesoft/nebula/client/storage/StorageClientTest.java

```diff
@@ -360,4 +366,161 @@ public void testScanEdgeWithAllCols() {
+            sslClient.connect();
+
+            ScanVertexResultIterator resultIterator = sslClient.scanVertex(
+                    "testStorageCA",
+                    "person");
+            while (resultIterator.hasNext()) {
+                ScanVertexResult result = null;
+                try {
+                    result = resultIterator.next();
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    assert (false);
+                }
+                if (result.isEmpty()) {
+                    continue;
+                }
+                Assert.assertEquals(1, result.getPropNames().size());
+                assert (result.getPropNames().get(0).equals("_vid"));
+                assert (result.isAllSuccess());
+
+                List<VertexRow> rows = result.getVertices();
+                for (VertexRow row : rows) {
+                    try {
+                        assert (Arrays.asList("1", "2", "3", "4", "5")
+                                .contains(row.getVid().asString()));
+                    } catch (UnsupportedEncodingException e) {
+                        e.printStackTrace();
+                        assert (false);
+                    }
+                    assert (row.getProps().size() == 0);
+                }
+
+                List<VertexTableRow> tableRows = result.getVertexTableRows();
+                for (VertexTableRow tableRow : tableRows) {
+                    try {
+                        assert (Arrays.asList("1", "2", "3", "4", "5")
+                                .contains(tableRow.getVid().asString()));
+                        assert (Arrays.asList("1", "2", "3", "4", "5")
+                                .contains(tableRow.getString(0)));
+                    } catch (UnsupportedEncodingException e) {
+                        e.printStackTrace();
+                        assert (false);
+                    }
+                }
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+            assert (false);
+        } finally {
+            if (sslClient != null) {
+                try {
+                    sslClient.close();
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+    }
+
+
+    @Test
+    public void testSelfSignedSSL() {
+        // start nebula service with ssl enable
+        List<HostAddress> address = null;
+        StorageClient sslClient = null;
+        Runtime runtime = Runtime.getRuntime();
+        try {
+
+            address = Arrays.asList(new HostAddress(ip, 8559));
+
+            // mock graph data
+            MockStorageData.mockSelfSslData();
+
+            SSLParam sslParam = new SelfSignedSSLParam(
+                    "src/test/resources/ssl/selfsigned.pem",
+                    "src/test/resources/ssl/selfsigned.key",
+                    "vesoft");
```

Comment:
According to the doc the user should provide the path to the password instead of the plain text, so the implementation of `SelfSignedSSLParam()` should be modified.

Reply:
keep the password as String.